traffic_ops/app/bin/checks/DnssecRefresh/config/config.go

```diff
@@ -0,0 +1,153 @@
+package config
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import (
+	"crypto/tls"
+	"errors"
+	"fmt"
+	"github.com/apache/trafficcontrol/lib/go-log"
+	"github.com/pborman/getopt/v2"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+)
+
+type Creds struct {
+	User     string `json:"u"`
+	Password string `json:"p"`
+}
+
+type Cfg struct {
+	LogLocationErr   string
+	LogLocationInfo  string
+	LogLocationWarn  string
+	LogLocationDebug string
+	TOInsecure       bool
+	TOUser           string
+	TOPass           string
+	TOUrl            string
+	Transport        *http.Transport
+}
+
+type ToResponse struct {
+	Response string `json:"response"`
+}
+
+func Dclose(c io.Closer) {
+	if err := c.Close(); err != nil {
```

Comment:
Should this log the error? That'd be a reason to keep it (vs as above, `defer body.Close()`)

Reply:
yes it does log an error now (as noted above).